libraries/common/subscriptions/helpers/buildRegisterUrl.spec.js

```diff
@@ -0,0 +1,56 @@
+import queryString from 'query-string';
+import { LEGACY_URL } from '../../constants/Registration';
+import { CHECKOUT_PATH, INDEX_PATH } from '../../constants/RoutePaths';
+import buildRegisterUrl from './buildRegisterUrl';
+
+describe('buildRegisterUrl', () => {
+  it('should create the expected url when no redirect was passed', () => {
+    const url = LEGACY_URL;
```

Comment:
Why do you store this variable? It's not used further on. Please put its value inline.
Please: also for the rest of this test.

Reply:
i thought it would make the test more readable. so at each test you know which parameters are passed to `buildRegisterUrl` and which value they have.

but i've updated the tests like you requested.